src/local-peers.js

```diff
@@ -517,30 +470,38 @@ export class LocalPeers extends TypedEmitter {
     switch (type) {
       case 'Invite': {
         const invite = Invite.decode(value)
-        assertInviteHasKeys(invite)
+        if (!isInviteValid(invite)) {
+          this.#l.log('Received invalid invite')
+          return
```

Comment:
I think we should throw here, since `Invite.decode()` will also throw for invalid invites (e.g. can't parse the protobuf), which makes me realise that we need to catch unhandled errors when calling this within `onmessage` of protomux.

Reply:
Made this throw in 572d09cf9d030596f3b37a5ea4542321e629bb76, which required catching the error in `onmessage` as you mentioned. Filed #514 in case there's more we want to do around this in general.